app/src/main/java/org/standardnotes/notes/MainActivity.kt

```diff
@@ -1,21 +1,37 @@
 package org.standardnotes.notes
 
 import android.content.Intent
+import android.content.SharedPreferences
 import android.os.Bundle
+import android.preference.PreferenceManager
 import android.support.v7.app.AppCompatActivity
 import android.support.v7.widget.Toolbar
 import android.view.Menu
 import android.view.MenuItem
 import android.view.WindowManager
+import android.widget.Toast
 import kotlinx.android.synthetic.main.activity_main.*
 import org.standardnotes.notes.frag.NoteListFragment
 
 class MainActivity : AppCompatActivity() {
 
+    companion object {
+        const val IS_SCREENSHOTTING_ENABLED = "toggle_screenshots"
+    }
+    lateinit var prefs: SharedPreferences
+    lateinit var editor: SharedPreferences.Editor
+
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
-        window.setFlags(WindowManager.LayoutParams.FLAG_SECURE,
-                WindowManager.LayoutParams.FLAG_SECURE)
+
+        prefs = PreferenceManager.getDefaultSharedPreferences(this)
+        editor = prefs.edit()
+
+        if (!isScreenshottingEnabled()) {
+            window.setFlags(WindowManager.LayoutParams.FLAG_SECURE,
+                    WindowManager.LayoutParams.FLAG_SECURE)
+        }
+
```

Comment:
If this is going to happen in every Activity, we should probably create a base Activity class.